### DISCUSSION TITLE
Welcome to ghz Discussions!

### DISCUSSION THREAD

**Discussion by bojand** [bojand/ghz#249] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by glf1990** [comment]
will  this tool  support to upload the  response cost    to Prometheus   during the  stress testing  time

**Comment by lserranov** [comment]
I saw some ghz examples and in those the payload is hardcoded. Can we  have the payload coming from a CSV file or something? That way we could send payload representing different users to the server.

As example I want payload to be  "user_id": "172465689" but in the next request I want a different user_id.

Thanks!

**Comment by YuvalAbadi** [comment]
Hi 

I am new to use GHZ.

i use 2VM with 4cpu  each ,
run ghz  on one machine to test server on other macine
1) ghz take 390-250% cpu while   the server is only at 70-90% cpu.
i think GHZ  not capable to test the server?
2) when i run GHZ with -z xx, for duration test,  it alwayes ends with some error.  (-z 1s )
3) run GHZ with -n xx  c xx ends very fast,  when I increase n to larger number, the test is killed after a while,


is any one know how much load GHZ capable to build?
how to make long duration test with out errors?  ( -z 1 got errors)

Thanks
Yuval

**Comment by abbhar** [comment]
hi @bojand, 
i found it very useful and would like to contribute to it, can you point me to how to go about it. 
if you have any design document for this, that would be great.
